Packs/NTT_Cyber_Threat_Sensor/Integrations/fetch_incidents/README.md

```diff
@@ -0,0 +1,100 @@
+Retrieve alerts and recommendations from NTT CTS
+This integration was integrated and tested with version xx of NTT Cyber Threat Sensor
+## Configure NTT Cyber Threat Sensor on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for NTT Cyber Threat Sensor.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| APIKEY | The API key for accessing CTS over AWS  | True |
+| TENANT_ID | Tenant identification. UUID formatted string | True |
+| DAYS_BACK | Days to fetch for the first time this application runs | True |
+| ITEMS_TO_FETCH | Number of items to fetch each iteration \(1 to 100\) | True |
+| SOARTOKEN | The unique key for accessing the alerts and active response recommendations | True |
+| isFetch | Fetch incidents | False |
+| incidentType | Incident type | False |
+| BASEURL | The base URL for the backend to consume from | True |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+### fetch_incidents
+***
+Retrieve incidents since last poll
+
+
+#### Base Command
+
+`fetch_incidents`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| CTS.DeviceName | string | Name of detecting device | 
+| CTS.Confidence | string | Confidence level | 
+| CTS.Severity | string | Severity level | 
+| CTS.ShortDescription | string | Short description of incident | 
+| CTS.ARAId | string | Id of this Active Response API | 
+| CTS.ARADetails | unknown | Details of the recommended actions | 
+| CTS.Signature | unknown | Signature name of detection | 
+| CTS.OccuredTime | unknown | Date and time of occurrence or detection | 
+| CTS.DeviceId | unknown | Id of detecting device | 
+| CTS.TenantId | unknown | Id of the tenant | 
+| CTS.EventID | string | Id of the event | 
+
+
+#### Command Example
+```!fetch_incidents ```
```

Comment:
Please create a new README file after you remove the `fetch-incidents` command - 
It shouldn't be here :)